### PR TITLE
Bug Fix - batch reverse order

### PR DESCRIPTION
**Pull request by HanJiyao**
This PR changes to reverse order to bathc command

### CODE REVIEW

File: src/main/java/seedu/contax/logic/commands/BatchCommand.java [hunk 1]
```diff
@@ -85,11 +85,11 @@ public CommandResult execute(Model model) throws CommandException {
         }
         List<CommandResult> commandResultList = new ArrayList<>();
         Person restorePerson = model.getFilteredPersonList().get(indexList.get(0).getZeroBased());
-        for (Index index: indexList) {
+        for (int i = indexList.get(indexList.size() - 1).getOneBased(); i >= indexList.get(0).getOneBased(); i--) {
```
Comment (sharpstorm) [inline src/main/java/seedu/contax/logic/commands/BatchCommand.java:88]: This loop does not look right, it is running on continuous indexes (which may not be the case in batch).

**Review by sharpstorm — CHANGES_REQUESTED** [review]
Please check the logic of the command.

File: src/main/java/seedu/contax/logic/commands/BatchCommand.java (hunk 1 above)
Comment (HanJiyao) [reply src/main/java/seedu/contax/logic/commands/BatchCommand.java:88]: Ohh my careless, thx changed

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm

**Pull request merged**